modules/RTC/JitsiStreamBlurEffect.js

```diff
@@ -0,0 +1,183 @@
+import JitsiStreamEffect from './JitsiStreamEffect';
+import { getLogger } from 'jitsi-meet-logger';
+import { load, drawBokehEffect } from '@tensorflow-models/body-pix';
+
+const logger = getLogger(__filename);
+
+const bpModelPromise = load(0.5);
+
+/**
+ * Represents a modified MediaStream that adds blur to video background.
+ * <tt>JitsiStreamBlurEffect</tt> does the processing of the original
+ * video stream.
+ */
+export default class JitsiStreamBlurEffect extends JitsiStreamEffect {
+
+    /**
+     * Represents a modified video MediaStream track.
+     * @constructor
+     * @param stream
+     */
+    constructor(stream) {
+
+        super(stream);
+
+        let n = 1;
+
+        // Original video stream settings
+        const firstVideoTrack = this._stream.getVideoTracks().shift();
+        const { height, frameRate, width } = firstVideoTrack.getSettings();
```

Comment:
What happens if the browser doesn't support `getSettings`?

Reply:
I will use getContraints as alternative but saw in RTC.js _newCreateLocalTracks the following call :         const { deviceId, facingMode } = track.getSettings();